Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -93,5 +93,42 @@ TEST_F(SerializerTest, EncodesNullProtoToBytes) {
   ExpectRoundTrip(proto, bytes, FieldValue::Type::Null);
 }
 
+TEST_F(SerializerTest, EncodesBoolModelToProto) {
+  for (bool test : {true, false}) {
+    FieldValue model = FieldValue::BooleanValue(test);
+    Serializer::TypedValue proto{FieldValue::Type::Boolean,
+                                 google_firestore_v1beta1_Value_init_default};
+    proto.value.boolean_value = test;
+    ExpectRoundTrip(model, proto, FieldValue::Type::Boolean);
+  }
+}
+
+TEST_F(SerializerTest, EncodesBoolProtoToBytes) {
+  struct TestCase {
+    bool value;
+    std::vector<uint8_t> bytes;
+  };
+
+  /* NB: proto true_bytes were created via (and similarly for false_bytes):
```

Comment:
This comment is great, but a little repetitive with the one for nulls. Perhaps move the command to a file-level comment (along the lines of _byte values used in tests were obtained by running echo 'APPROPRIATE_VALUE_HERE' | `<`the rest of the command`>`_), and then only specify what was `echo`ed?

Reply:
Done.